examples/CreatePerTitleEncodingDashAndHls.php

```diff
@@ -0,0 +1,398 @@
+ * the generation of the Per-Title.
+ *
+ * @param ApiClient $apiClient The Bitmovin Api-Client
+ * @param Encoding $encoding The reference of the encoding
+ * @param S3Output $s3Output The output the files should be written to
+ * @param Stream $stream The Per-Title template video stream
+ * @param String $outputPathSuffix The suffix to be concatenated to OUTPUT_BASE_PATH to compose the output path of the muxing.
+ * @return FMP4Muxing
+ * @throws BitmovinException
+ */
+function createfMp4Muxing($apiClient, $encoding, $s3Output, $stream, $outputPathSuffix)
+{
+    $acl = new Acl(AclPermission::ACL_PUBLIC_READ);
+    $muxingOutput = new EncodingOutput($s3Output);
+    $muxingOutput->setOutputPath(OUTPUT_BASE_PATH . $outputPathSuffix);
+    $muxingOutput->setAcl(array($acl));
+
+    $muxingStream = new MuxingStream();
+    $muxingStream->setStreamId($stream->getId());
+
+    $fmp4Muxing = new FMP4Muxing();
+    $fmp4Muxing->setStreams(array($muxingStream));
+    $fmp4Muxing->setSegmentLength(4);
+    $fmp4Muxing->setSegmentNaming("seg_%number%.m4s");
+    $fmp4Muxing->setInitSegmentName("init.mp4");
+    $fmp4Muxing->setOutputs(array($muxingOutput));
+
+    return $apiClient->encodings()->muxings($encoding)->fmp4Muxing()->create($fmp4Muxing);
+}
+
+/**
+ * A TS muxing will be created for with the Per-Title video stream template and the audio stream.
+ * This muxing must define either {uuid} or {bitrate} in the output path.  These placeholders will be replaced during
+ * the generation of the Per-Title.
+ *
+ * @param ApiClient $apiClient The Bitmovin Api-Client
+ * @param Encoding $encoding The reference of the encoding
+ * @param S3Output $s3Output The output the files should be written to
+ * @param Stream $stream The Per-Title template video stream
+ * @param String $outputPathSuffix The suffix to be concatenated to OUTPUT_BASE_PATH to compose the output path of the muxing.
+ * @return TSMuxing
+ * @throws BitmovinException
+ */
+function createTSMuxing($apiClient, $encoding, $s3Output, $stream, $outputPathSuffix)
+{
+    $acl = new Acl(AclPermission::ACL_PUBLIC_READ);
+    $muxingOutput = new EncodingOutput($s3Output);
+    $muxingOutput->setOutputPath(OUTPUT_BASE_PATH . $outputPathSuffix);
+    $muxingOutput->setAcl(array($acl));
+
+    $muxingStream = new MuxingStream();
+    $muxingStream->setStreamId($stream->getId());
+
+    $tsMuxing = new TSMuxing();
+    $tsMuxing->setStreams(array($muxingStream));
+    $tsMuxing->setSegmentLength(4);
+    $tsMuxing->setSegmentNaming("segment_%number%.ts");
+    $tsMuxing->setOutputs(array($muxingOutput));
+
+    return $apiClient->encodings()->muxings($encoding)->tsMuxing()->create($tsMuxing);
+}
+
+
+
+/**
+ * The encoding will be started with the per title object and the auto representations set. If the auto
+ * representation is set, stream configurations will be automatically added to the Per-Title profile. In that case
```

Comment:
As far as I know, `AutoRepresentation` is a mandatory configuration part for Per-Title in general, even if its empty.